package.json

```diff
@@ -16,7 +16,7 @@
     "build:clean": "yarn clean && yarn build",
     "build:docs": "yarn workspaces foreach --all --exclude @ocap/monorepo --exclude @ocap/extension --parallel --interlaced --verbose run build:docs",
     "build:source": "ts-bridge --project tsconfig.build.json --verbose",
-    "build:special": "yarn workspace @ocap/vite-plugins run build && yarn workspace @metamask/kernel-shims run build && yarn workspace @metamask/kernel-browser-runtime run build:vite && yarn workspace @ocap/extension run build && yarn workspace @ocap/kernel-test run build:vats",
+    "build:special": "yarn workspace @ocap/vite-plugins run build && yarn workspace @metamask/kernel-shims run build && yarn workspace @metamask/kernel-browser-runtime run build:vite && yarn workspace @metamask/kernel-ui run build:vite && yarn workspace @ocap/extension run build && yarn workspace @ocap/kernel-test run build:vats",
```

Comment:
Unrelated to this PR, but this script is becoming pretty unwieldy. `build:source` is written in the way it is to avoid invoking `ts-bridge` (and thereby `tsc`) for each package built with `ts-bridge`. We could add a `build:special` command (or something with a better name) to each package so we can do `yarn workspaces foreach ... run build:special` for the corresponding script here. Just leaving this here as food for thought.